### DISCUSSION TITLE
guys,  has the program been abandoned?

### DISCUSSION THREAD

**Discussion by 2426232528** [isar/isar#1682] [Help]
First of all, thanks to all the developers for their contributions. Now with the flutter version update, it causes a lot of conflicting issues with the v3 version. the v4 version never went live, is it going to be abandoned? Can the project creator reply positively to this question? If it is abandoned, we will switch to other components, otherwise it will cause a lot of difficulties in updating the existing project. If it's not abandoned, can you reply clearly about the update plan? This is an excellent project, and I like it very much. Thank you very much!

**Comment by spatialbits** [comment]
Maybe? Probably? There are other similar discussions going on (eg https://github.com/isar/isar/discussions/118) and there is https://github.com/isar-community.

**Comment by jason-downs** [comment]
Sadly I think so...I've switched out Isar for Drift in my flutter project and that package is still very-well maintained as of now.

**Reply by MrCsabaToth** [reply]
Drift is built on top of SQLite = it'll never reach the speed of isar. Before isar I used Floor, Moor is also SQLite based. Doggone slow. I'm not gonna go back there.
